docs/usage/check.rst.inc

```diff
@@ -85,75 +85,109 @@ borg check
 Description
 ~~~~~~~~~~~
 
-The check command verifies the consistency of a repository and the corresponding archives.
-
-check --repair is a potentially dangerous function and might lead to data loss
-(for kinds of corruption it is not capable of dealing with). BE VERY CAREFUL!
+The check command verifies the consistency of a repository and its archives.
```

Comment:
thanks for the PR, but this is the wrong file - see the comment in line 1 of this file.

likely this needs to be changed in:
- master branch: src/borg/archiver/check_cmd.py
- 1.2-maint branch: src/borg/archiver.py

also, usually smaller incremental changes are preferred. do you think this needs a complete rewrite?

Reply:
> but this is the wrong file - see the comment in line 1 of this file.

:unamused: 

I definitely didn't expect this when there's a distinct "View this on GitHub" link on the website... I'll look into it and make the changes where they are expected. Surely no big deal.

I just checked `src/borg/archiver/check_cmd.py` and found it. Even though I'm a bit reluctant to add everything to the `--help` epilog (IMHO `--help` should cover the important bits, details should go into the man page), I'll do it this way. So, after checking `create_cmd.py` I really just copy everything to the argparse epilog? :thinking: 

> also, usually smaller incremental changes are preferred. do you think this needs a complete rewrite?

Yes, it needs a complete rewrite, simply because the current docs are way to tech-centered to be understandable by an ordinary user (especially non-developers). I'm not even sure whether my rewrite is user-centered enough...

Anyway, even though the previous contents of the docs were mostly reused from a contents perspective, the sentences and the order of the information is totally different - and it's important that it is different. For example, everything that was in this long list of "`borg check` does this, then this, and next …" was rewritten. These paragraphs surely explained what the *code* does, but an user doesn't care about what the code does, only about what it means. That's why I've created a distinct "About repair mode" section, simply because from an user's perspective and considering what repair mode does, repairing a repo is a whole different story. Thinking about it, `borg check --repair` shouldn't be a thing at all, there should rather be a distinct `borg repair` command...

So, no, unfortunately it's not possible to achieve this degree of a docs overhaul with incremental changes. It's no code after all, but human language: even though the contents are mostly the same, the sentences were written differently. Stuff was moved around and re-paraphrased to consider the new context. The new docs basically follow a different "golden thread": Instead of explaining what the code does, the docs now try to explain what it means running it.